go.mod

```diff
@@ -17,23 +17,25 @@ replace (
 
 require (
 	github.com/alexflint/go-arg v1.5.1
+	github.com/google/go-containerregistry v0.20.2
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
-	github.com/onsi/ginkgo/v2 v2.19.0
-	github.com/onsi/gomega v1.33.1
+	github.com/onsi/ginkgo/v2 v2.20.1
```

Comment:
please change versions in this file https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/automation/build-test.sh#L5 too